rpki-validator/src/main/java/net/ripe/rpki/validator3/domain/validation/CertificateTreeValidationService.java

```diff
@@ -271,7 +271,7 @@ private void validateCertificateAuthority(final TrustAnchor trustAnchor,
                 if (rpkiRepository.getStatus() == RpkiRepository.Status.FAILED) {
                     temporary.error(ValidationString.VALIDATOR_NO_MANIFEST_REPOSITORY_FAILED, rpkiRepository.getLocationUri());
                 } else {
-                    temporary.error(ValidationString.VALIDATOR_NO_LOCAL_MANIFEST_NO_MANIFEST_IN_REPOSITORY, rpkiRepository.getLocationUri());
+                    temporary.error(ValidationString.VALIDATOR_NO_LOCAL_MANIFEST_NO_MANIFEST_IN_REPOSITORY, manifestUri.toString(), rpkiRepository.getLocationUri());
```

Comment:
I think it requires change in rpki-commons as well. Is it already there?

Reply:
It already is in rpki-commons, that is how I found out:
![Screenshot 2020-02-28 at 15 37 53](https://user-images.githubusercontent.com/199058/75557458-55ac8d80-5a40-11ea-9e81-45613b243419.png)